driver/src/price_finding/optimization_price_finder.rs

```diff
@@ -247,13 +259,19 @@ fn write_input(input_file: &str, input: &str) -> std::io::Result<()> {
     Ok(())
 }
 
-fn run_solver(input_file: &str, result_folder: &str, solver_type: SolverType) -> Result<()> {
+fn run_solver(
+    input_file: &str,
+    result_folder: &str,
+    solver_type: SolverType,
+    solver_time_limit: u32,
```

Comment:
This would also allow to encapsulate the logic inside the `solver_type.to_args()` method